e2e_samples/parking_sensors/scripts/deploy_infrastructure.sh

```diff
@@ -239,22 +264,32 @@ KEYVAULT_RESOURCE_ID=$(echo "$arm_output" | jq -r '.properties.outputs.keyvault_
 
 ####################
 # DATA FACTORY
+databricks_folder_name="/Users/${kv_owner_name,,}"
+databricks_folder_name_standardize="${databricks_folder_name}/02_standardize.py"
+databricks_folder_name_transform="${databricks_folder_name}/03_transform.py"
+echo "databricks_folder_name_standardize: ${databricks_folder_name_standardize}"
+echo "databricks_folder_name_transform: ${databricks_folder_name_transform}"
 
 echo "Updating Data Factory LinkedService to point to newly deployed resources (KeyVault and DataLake)."
 # Create a copy of the ADF dir into a .tmp/ folder.
 adfTempDir=.tmp/adf
 mkdir -p $adfTempDir && cp -a adf/ .tmp/
 # Update ADF LinkedServices to point to newly deployed Datalake URL, KeyVault URL, and Databricks workspace URL
 tmpfile=.tmpfile
+tmpfile2=.tmpfile2
```

Comment:
you set this but then you neve use it. have you missed a reference to this variable (maybe in lines 286 and 287?)

Reply:
Yes, it was meant for a test. Is removed now! Tks!